lms/templates/courseware/course_updates.html

```diff
@@ -1,48 +1,18 @@
+<%page expression_filter="h"/>
```

Comment:
I wouldn't change this file as this is for the old UI. Let's target the changes only to the new template, course-updates-fragment.html.